R/set_seeds.R

```diff
@@ -0,0 +1,82 @@
+##' @title Set Seed for `mlrkeras`
+##' @description `set_seeds`: sets a seed in Random, Python, NumPy and Tensorflow.
```

Comment:
Please explain / link to more info on why parallel / gpu needs to be disabled etc.